eng/pipelines/ci.yml

```diff
@@ -128,6 +128,12 @@ jobs:
           popd
         displayName: 'Run Acceptance Tests'
 
+      - pwsh: |
+          pushd ./src
+          npm test
+          popd
+        displayName: 'Run Unit Tests'
```

Comment:
`popd` is unnecessary here as shell context doesn't carry from step to step. You could also configure the shell directory externally if you want:

```
      - pwsh: npm test
        displayName: 'Run Unit Tests'
        workingDirectory: $(System.DefaultWorkingDirectory)/src
```